.github/workflows/ci.yml

```diff
@@ -0,0 +1,104 @@
+name: Continuous integration
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
```

Comment:
The package doesn't seem to "officially" support 3.9 yet (according to `setup.py` and `.travis.yml`). I'm all for 3.9 support, but probably should stick to 3.8 for now.

Also applies to other similar lines in `ci.yml`, `release.yml` and `staging.yml`.

Reply:
Tangentially related: .travis.yml can probably be removed? https://app.travis-ci.com/github/City-of-Helsinki/django-orghierarchy

Reply:
^ @tuomas777 Any idea if Travis is relevant anymore?